src/elements/about-block.html

```diff
@@ -0,0 +1,170 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/iron-icon/iron-icon.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+<link rel="import" href="hoverboard-icons.html">
+
+<dom-module id="about-block">
+    <template>
+        <style is="custom-style" include="shared-styles flex flex-alignment flex-reverse"></style>
+
+        <style>
+             :host {
+                display: block;
+                position: relative;
+                color: var(--dark-primary-color);
+            }
+
+            .container {
+                margin: 48px 40px;
+            }
+
+            .description-block,
+            .statistics-block {
+                padding: 16px;
+            }
+
+            .statistics-block .row-1 {
+                margin-bottom: 16px;
```

Comment:
I don't like a solution in naming `row-1`, `row-2`. I recommend using pseudo classes, it's more abstract